pkg/securitypolicy/internal.go

```diff
@@ -0,0 +1,254 @@
+//go:build linux
+// +build linux
+
+package securitypolicy
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-cmp/cmp"
```

Comment:
This package says:

```
It is intended to only be used in tests, as performance is not a goal and it may panic if it cannot compare the values.
```

I would suggest we just implement the function we need, rather than pull in an external dependency.

Reply:
looks like string slice comparison is enough for non-test code.